### ISSUE TITLE
1.8.8 - Cannot Make Wither Skeleton

### ISSUE THREAD

**Issue by kangarko** [kangarko/Boss#51]
An attempt to do so results in the following error: 

```
[10:44:10 INFO]: Boss 1.5.8 encountered a NoSuchFieldError! Please check your error.log and report this issue with the information in that file.
[10:44:10 INFO]: java.lang.NoSuchFieldError: STRAY
[10:44:10 INFO]: 	at me.kangarko.boss.api.model.BossSpecificSetting$5.protected(SourceFile:476)
[10:44:10 INFO]: 	at me.kangarko.boss.final. .aUx.transient(SourceFile:166)
[10:44:10 INFO]: 	at me.kangarko.boss.final. .aUx.spawn(SourceFile:118)
[10:44:10 INFO]: 	at me.kangarko.boss.final. .at.onClick(SourceFile:140)
[10:44:10 INFO]: 	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[10:44:10 INFO]: 	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[10:44:10 INFO]: 	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[10:44:10 INFO]: 	at java.lang.reflect.Method.invoke(Method.java:498)
[10:44:10 INFO]: 	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306)
[10:44:10 INFO]: 	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[10:44:10 INFO]: 	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502)
[10:44:10 INFO]: 	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487)
[10:44:10 INFO]: *-----------------------------------------------*
[10:44:10 INFO]:  Error making a Boss!
[10:44:10 INFO]:  Entity: CraftSkeleton
[10:44:10 INFO]:  Boss: Skel
[10:44:10 INFO]:  NoSuchFieldError: STRAY
[10:44:10 INFO]: *-----------------------------------------------*
```

**Comment by kangarko** [comment]
Issue has been partially fixed, error is gone but the skeleton is not turned wither, and the Stray icon is still present but is not available in 1.8.x.

**Comment by kangarko** [comment]
Was fixed some time ago, I just forgot to close this.
